lib/network/modules/View.js

```diff
@@ -45,7 +44,7 @@ class View {
   /**
    * This function zooms out to fit all data on screen based on amount of nodes
    * @param {Object} [options={{nodes=Array}}]
```

Comment:
?? The default value is a hash of a hash in which a parameter is initialized to class Array?

Nononono....I won't buy that.

Reply:
With my insights on the jsdoc data types, Perhaps is meant: `[options={{nodes:[]}}]`.

In any case, the double brace notation is valid.